app/models/admin.rb

```diff
@@ -0,0 +1,6 @@
+class Admin < ApplicationRecord
+  
+  def self.order_by_name_reverse
```

Comment:
Admin doesn't map to a database table, so it shouldn't have a model. Instead, it's just being used as a namespace, or a tool to organize routes. This query could live in the Shelters model and be tested in the Shelters_spec